linux_os/guide/system/auditing/auditd_configure_rules/audit_file_deletion_events/audit_rules_file_deletion_events.rule

```diff
@@ -54,3 +54,14 @@ ocil: |-
     {{{ ocil_audit_syscall(syscall="renameat") }}}
 
 {{{ ocil_clause_entry_audit_syscall() }}}
+
+warnings:
+    - general: |-
+        This rule aims to attest collection of events from multiple syscalls
+        related to file deletion. It was written with DISA STIG in mind, other
```

Comment:
I'd use a semicolon instead of a comma because these are two related but otherwise independent sentences.